pandas/_version.py

```diff
@@ -113,7 +113,7 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
     """
     rootdirs = []
 
-    for i in range(3):
+    for _i in range(3):
```

Comment:
no

Reply:
Are you against it in general or in cases where the variable is clearly a counter?

Reply:
i dont particularly care (in fact in general i dislike 1-letter variable names).  But i dont think this is worth futzing with over hundreds of files

Reply:
I think `for _ in range(3)` is more standard, does that work?

Reply:
`for _ in range(3)` works for bugbear. Would it be nicer to simply replace all unused variables with `"_"`? If we have nested loops with unused variables or in general multiple `"_" `within in function, it would probably be good to use the prefix strategy to avoid upsetting mypy (the two unused variables might have different types).

Reply:
I think I'd suggest `_` where possible (that's what's already used in many parts of the codebase), and prefixing `_` if/where necessary